config/default.js

```diff
@@ -22,6 +22,9 @@ module.exports = {
       save: 'save',
       new: 'Save Revision',
     },
+    plugins: {
+      directory: [path.join(__dirname, '../input-plugins')],
```

Comment:
Why doesn't this work? What is the value of `plugins.directory` in an implementation that's pulling in the default configuration from Punchcard? Can we force a value here in `init`?